### DISCUSSION TITLE
Retornos

### DISCUSSION THREAD

**Discussion by tavaresgerson** [BrasilAPI/BrasilAPI#353] [General]
Olá comunidade,

Primeiramente, excelente trabalho essa API (simplesmente fantástico). Mas, algo que fica muito subjetivo no meu ponto de vista é: "uma API feita por brasileiros, para brasileiros" porém, qual o sentido das mensagens de erro serem em inglês?! kk

Abraços!

**Comment by lucianopf** [comment]
Bom dia mestre, ótimo ponto hein :star_struck: !! Me fez refletir um bocado aqui! :thinking: 

Mas minha na opinião tem alguns motivos:

1. Apesar de ter dados referentes ao Brasil não quer dizer que quem irá utilizar é brasileiro. Veja como o Stripe por exemplo chegando no Brasil e implementando soluções no Brasil, imagina que eles tão implementando o checkout em que eles precisam validar o `zipcode`, utilizar o BrasilAPI seria uma boa alternativa pra eles, mas todos os sistemas deles são em inglês então misturar as coisas do lado dos sistemas deles deve atrapalhar um pouco :thinking: :grimacing: 

2. O projeto além de ter como objetivo servir como uma API útil pra consumidores tbm tem como objetivo de ser um local de aprendizado e "treinamento" pro mundo real, onde vc possivelmente terá que programar em inglês :eyes: 

Não sei se são argumentos válidos ou na vdd eu só to querendo provar meu viés de sempre codar em inglês :thinking: :joy: :man_facepalming: 

Será que fazem sentido ou to viajando? kkkk

Bora elaborar essa discussão galera! :pray:  :heart_eyes:

**Reply by RodriAndreotti** [reply]
Eu também prefiro tudo em Inglês, e também concordo com seus pontos @lucianopf.


Acho que essa discussão poderia ser feita juntamente àquela dos nomes de propriedades e endpoints (que por sinal também tem ótimos argumentos a favor do idioma brasuca... rs) que está rolando por aqui e no Slack, o que acha?

PR's e links onde já foi discutido isso:
https://github.com/BrasilAPI/BrasilAPI/pull/268
https://github.com/BrasilAPI/BrasilAPI/discussions/223#discussioncomment-350452
https://brasilapi.slack.com/archives/C01JW0ZD48K/p1632408224010400
https://brasilapi.slack.com/archives/C01JW0ZD48K/p1632236423002900

**Reply by tavaresgerson** [reply]
Obrigado @RodriAndreotti, irei apagar essa discussão, não vejo necessidade de levantar um questionando quando este já foi concluído. :-)

**Reply by RodriAndreotti** [reply]
Não sei se já foi concluído, kkkk
Mas se quiser votar lá no slack seu voto é bem vindo... rs

**Comment by tavaresgerson** [comment]
Compreendo seu ponto de vista: entidades de banco, controllers, etc. defino todos em inglês, até para ter um trabalho mais "fluído" e alguns ORM entender os relacionamentos mais fácil (como o CakePHP por exemplo :-D). Mas, o fato de ser mencionando "de brasileiros para brasileiros" faz todo o sentido um sistema voltado pra gente ué. :-)
